### PR TITLE
Add "Coin Wallet.app" v3.0.1

### PR DESCRIPTION
**Pull request by mahnunchik**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md). The same name used for snap https://snapcraft.io/coin
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/coin.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+cask "coin" do
+  version "3.0.1"
+  sha256 "ee0cee1c90b0431a059e49a8aabf678dc1243c833c4891f635043a76ab1a2489"
+
+  # github.com/CoinSpace/CoinSpace was verified as official when first introduced to the cask
```
Comment (miccal) [inline Casks/coin.rb:5]:
```suggestion
  # github.com/CoinSpace/CoinSpace/ was verified as official when first introduced to the cask
```

File: Casks/coin.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  url "https://github.com/CoinSpace/CoinSpace/releases/download/v#{version}/Coin.Wallet.dmg"
+  appcast "https://github.com/CoinSpace/CoinSpace/releases.atom"
+  name "Coin Wallet"
+  desc "Digital Currency Wallet"
```
Comment (miccal) [inline Casks/coin.rb:9]:
```suggestion
  desc "Digital currency wallet"
```

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @mahnunchik.

**Review by miccal — DISMISSED** [review]

**Comment by miccal** [comment]
Please rename the file name to `coin-wallet.rb` and the cask name to `coin-wallet` to follow the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md), thanks.

**Comment by mahnunchik** [comment]
@miccal is it ok that `coin-wallet` name will be different from linux `coin` snap and binary?

**Comment by miccal** [comment]
> @miccal is it ok that `coin-wallet` name will be different from linux `coin` snap and binary?

Yes, we follow the token reference more-or-less without exception.

**Comment by miccal** [comment]
Thank you @mahnunchik.

**Review by miccal — APPROVED** [review]

**Pull request merged**